src/pages/Bets/Bets_Logic.tsx

```diff
@@ -15,7 +15,7 @@ const BetsLogics = () => {
   const [myBetsEnabled, setMyBetsEnabled] = useState(true);
   const [isModalOpen, setIsModalOpen] = useState(false);
   const [selectedBet, setSelectedBet] = useState<BetHistory>();
-
+  const [selectedFilter, setSelectedFilter] = useState<string>("ALL_BETS");
```

Comment:
import `FilterOption` and use that as the type for this - `string` is v open to dev error for this